integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -66,6 +66,13 @@ if [ "${KATA_HYPERVISOR:-}" == "cloud-hypervisor" ]; then
 	oom_issue="https://github.com/kata-containers/tests/issues/2864"
 	info "$KATA_HYPERVISOR is failing on:"
 	info "pod oom: ${oom_issue}"
+
+elif [ "${arch}" == "aarch64" ]; then
```

Comment:
Hmm, actually, it might be safer to check if the arch is *not* `aarch64` and if that is true, do `K8S_TEST_UNION+=("k8s-oom.bats")`. That avoids aarch64 getting out-of-sync if new tests are added and someone forgets to update *both* these blocks.